brave/common/workers/worker_bindings.cc

```diff
@@ -80,10 +79,12 @@ void WorkerBindings::AddRoutes() {
   v8::Isolate* isolate = v8_context->GetIsolate();
 
   // onmessage handler
-  SetProperty(v8_context, v8_context->Global(),
-      v8::String::NewFromUtf8(isolate, "onmessage",
-          v8::NewStringType::kNormal).ToLocalChecked(),
+  auto maybe_set = v8_context->Global()->CreateDataProperty(
+      v8_context,
+      v8::String::NewFromUtf8(isolate, "onmessage", v8::NewStringType::kNormal)
+          .ToLocalChecked(),
       v8::Null(isolate));
+  DCHECK(maybe_set.IsJust() && maybe_set.FromJust());
```

Comment:
it is unlikely to fail so that is probably why we didn't check. I was just saying this DCHECK condition is actually opposite. Maybe we can `ThrowException` to notify the caller

Reply:
ah sorry, the original condition is correct.